SemiRP/Utils/AdminUtils/AdminHelper.cs

```diff
@@ -183,30 +183,47 @@ public static void VehicleDestroy(Player sender, int id = -1)
             if (!sender.AccountData.HavePerm("admin.cmds.vehicule.destroy"))
                 throw new Exception();
 
-
             Vehicle vehicle = CmdHelper.GetCurrentVehicleOrID(sender, id);
 
             Helper.DestroyVehicle(vehicle);
         }
 
-        public static Vehicle VehicleTpTo(Player sender)
+        public static Vehicle VehicleTpNearest(Player sender)
         {
-            if (!sender.AccountData.HavePerm("admin.cmds.vehicule.tp"))
+            if (!sender.AccountData.HavePerm("admin.cmds.vehicule.tpn"))
                 throw new Exception();
 
             var nearestVeh = Helper.GetNearestVehicle(sender);
 
             if (nearestVeh == null)
                 throw new Exception();
 
-            if (nearestVeh.Driver != null)
-                sender.Position = new Vector3(nearestVeh.Position.X, nearestVeh.Position.Y, nearestVeh.Position.Z + 1f);
-            else
-                sender.PutInVehicle(nearestVeh);
-
+            VehicleTp(sender, nearestVeh);
             return nearestVeh;
         }
 
+        public static Vehicle VehicleTpId(Player sender, int id)
+        {
+            if (!sender.AccountData.HavePerm("admin.cmds.vehicule.tpid"))
+                throw new Exception();
+
+            var vehicle = (Vehicle)Vehicle.Find(id);
```

Comment:
No need to cast here, use Vehicle type directly
```suggestion
            Vehicle vehicle = Vehicle.Find(id);
```